crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -679,6 +683,12 @@ impl VertexFormatSize for wgpu::VertexFormat {
 
 /// Contains an array where each entry describes a property of a single vertex.
 /// Matches the [`VertexFormats`](VertexFormat).
+///
+/// Be aware that the [`Float64`](VertexAttributeValues::Float64),
+/// [`Float64x2`](VertexAttributeValues::Float64x2), [`Float64x3`](VertexAttributeValues::Float64x3) and
+/// [`Float64x4`](VertexAttributeValues::Float64x4) need to be used with Caution, because they might cause
+/// Performance Problems due to [f64](std::primitive::f64) being between [considerably](https://groups.google.com/g/amgcl/c/Bz_X2B0UBWE)
```

Comment:
Looks like the word "between" shouldn't be here.

Reply:
"Caution" should probably be lowercase.

Reply:
Will fix it real quick, thank you :)